freezeyt/cli.py

```diff
@@ -89,6 +93,11 @@ def main(
         # --progress=bar.
         config.setdefault(
             'plugins', []).append('freezeyt.progressbar:LogPlugin')
+    
+    if (cleanup == None and config.get("cleanup") == None) or cleanup == True:
+        config['cleanup'] = True
+    elif cleanup == False:
+        config['cleanup'] = False
```

Comment:
If I am right that we want CLI as highest priority, you can simplified these conditions as we do with `prefix` see line 69 and lines 48 - 64 how `config` *object* is created. I supposed there is no need to create key `cleanup` in all cases. The `config` without the key is OK also. See the code in `freezer.py` on line 309

Reply:
many thanks @nappex for your help, I incorporated all of your comments